scripts/test_swe_bench_runner.py

```
@@ -0,0 +1,368 @@
+class TestAppendPrediction(unittest.TestCase):
+    """Tests for append_prediction."""
+
+    def test_writes_correct_schema(self):
+        with tempfile.NamedTemporaryFile(suffix=".jsonl", delete=False) as f:
+            path = f.name
+        try:
+            swe_bench_runner.append_prediction(path, "test__inst-1", "tiny-agent", "some diff")
+            swe_bench_runner.append_prediction(path, "test__inst-2", "tiny-agent", None)
+
+            with open(path) as fh:
+                lines = [json.loads(l) for l in fh if l.strip()]
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)